src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball1 = new Ball();
+        return ball1;
+    }
```

Comment:
[Don’t create temporary variables if you can directly return value from the method](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method)